hazelcast/src/test/java/com/hazelcast/test/HazelcastParallelClassRunner.java

```diff
@@ -149,12 +149,7 @@ public void run() {
 
         private final Properties globalProperties;
 
-        private final ThreadLocal<Properties> localProperties = new InheritableThreadLocal<Properties>() {
-            @Override
-            protected Properties initialValue() {
-                return init(new Properties());
-            }
-        };
+        private final ThreadLocal<Properties> localProperties = new ThreadLocal<Properties>();
```

Comment:
Hmm, you might be right that we don't need it here. But I don't like to have this pattern in the code anymore, so no one can pick it up accidentally.